pkg/operator/ceph/csi/spec.go

```diff
@@ -143,8 +143,8 @@ var (
 	DefaultRegistrarImage   = "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.10.0"
 	DefaultProvisionerImage = "registry.k8s.io/sig-storage/csi-provisioner:v4.0.0"
 	DefaultAttacherImage    = "registry.k8s.io/sig-storage/csi-attacher:v4.5.0"
-	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v7.0.1"
-	DefaultResizerImage     = "registry.k8s.io/sig-storage/csi-resizer:v1.10.0"
+	DefaultSnapshotterImage = "registry.k8s.io/sig-storage/csi-snapshotter:v6.3.2"
```

Comment:
Was it intentional to downgrade these image versions?

Reply:
removed this change.